src/main/java/oncall/view/OutputView.java

```diff
@@ -0,0 +1,45 @@
+package oncall.view;
+
+import java.time.DayOfWeek;
+import java.time.format.TextStyle;
+import java.util.List;
+import java.util.Locale;
+import oncall.domain.Workers;
+import oncall.domain.constants.CustomDayOfWeek;
+import oncall.domain.constants.Holiday;
+import oncall.domain.constants.Month;
+import oncall.view.console.ConsoleWriter;
+
+public class OutputView {
+    private static final String MONTH = "월";
+    private static final String DAY = "일";
```

Comment:
`MONTH`, `DAY`가 한글 문자인 것을 모르고, 아래 29-30번 줄에서 혼란스러웠습니다, `DateTimeFormatter`를 통해서 포매팅해보시는 것도 좋을 것 같아요!
```suggestion
private static final DateTimeFormatter FORMAT_PATTERN = DateTimeFormatter.ofPattern("M월 d일");
```

Reply:
아하 .. 좋은 라이브러리 알려주셔서 감사합니다.
자바에 대한 이해가 깊으신 것 같아요 !